docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -165,21 +192,39 @@ insert into users (email, name, state) values ('sue@help.com', 'Sue Sas', 'CA');
 insert into users (email, name, state) values ('tom@yes.com', 'Tom and Jerry', 'NV');
 ```
 
-Copy the cql file to the k8ssandra container (pod) :
+Copy the cql file to the k8ssandra container (pod):
 
 `kubectl cp test_data.cql k8ssandra-dc1-default-sts-0:/tmp -c cassandra`
 
-Add the data to the Cassandra database:
+Before you can launch the CQLSH instance that's deployed by K8ssandra in your Kubernetes cluster, you'll need authentication credentials. By default, the username secret for K8ssandra is `k8ssandra-superuser`. To verify, you can extract and decode the username secret:
```

Comment:
A couple things here. First, I would change `username secret` to `superuser secret`. Secondly it defaults to `<cluster-name>-superuser`. The cluster name defaults to the release name.  Let's say you install as follows:

```
$ helm install demo k8ssandra/k8ssandra
```
The release name is `demo` which means the cluster name will also default to `demo`. The superuser secret will then be named `demo-superuser`.

Reply:
Incorporated comments from John Sanda. Please check that with cluster-name of demo, all corresponding changes are correct in examples.